src/UnitTests/GitHub.Exports/VSServicesTests.cs

```diff
@@ -1,13 +1,87 @@
 ﻿using System;
+using System.IO;
+using System.Runtime.InteropServices;
 using GitHub.Services;
-using Microsoft.TeamFoundation.Git.Controls.Extensibility;
 using NSubstitute;
 using Xunit;
+using DTE = EnvDTE.DTE;
+using Rothko;
 
 public class VSServicesTests
 {
+    public class TryOpenRepository : TestBaseClass
```

Comment:
Following the convention used in the rest of the tests, this should be called `TheTryOpenRepositoryMethod`.